journal/modernized/01-journal.adoc

```diff
@@ -385,7 +394,7 @@ I found that all the holy men of God used that language,
 and Christ taught his disciples to pray in that language; "`Our Father,
 which art in heaven, hallowed be thy name, thy kingdom come,
 thy will be done on earth as it is in heaven.`"
-Mat, vi.
+Mat. 6.
```

Comment:
My scripture reference finder struggled here because it was only a reference to a chapter, not chapter and verse. Can you also change it to be `Matt. 6` instead, since it would have normalized to that if the finder had caught it.